maxim/tests/test_test_runs.py

```
@@ -241,5 +263,597 @@ def error(self, message: str, e: Optional[Exception] = None) -> None:
+            print(
+                f"Test run completed with {len(result.failed_entry_indices)} failed entries"
+            )
+
+    def test_create_test_run_with_image_variables_prompt_workflow_and_local_evaluators(
+        self,
+    ):
+        """Test prompt workflow with image variable columns and local evaluators"""
+
+        class MyCustomEvaluator(BaseEvaluator):
+            def evaluate(
+                self, result: LocalEvaluatorResultParameter, data: LocalData
```

Ruff (0.11.9)

Unused method argument: `result`

(ARG002)